web/src/components/molecules/Common/Form/AssetItem/index.tsx

```diff
@@ -141,4 +154,27 @@ const AssetButton = styled(Button)`
   border: 1px dashed #d9d9d9;
 `;
 
+const AssetWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+`;
+
+const AssetIcon = styled(Button)`
+  color: #000000d9;
+  margin-top: 4px;
+  top: 3px;
```

Comment:
Since it isn't aligned relatively to anything, I don't think the top: 3px will do anything

Reply:
@KaWaite , it is centered by the flex wrapper, but the SVG image itself has 3px issue.